GVRf/Framework/src/org/gearvrf/GVRRenderTexture.java

```diff
@@ -55,6 +55,20 @@ public GVRRenderTexture(GVRContext gvrContext, int width, int height,
     GVRRenderTexture(GVRContext gvrContext, long ptr) {
         super(gvrContext, ptr);
     }
+    
+    /**
+     * Return the width of GVRRenderTexture (FBO)
+     */
+    public int getWidth() {
+    	return NativeRenderTexture.getWidth(getPtr());
+    }
+
```

Comment:
My understanding is that GVRF is designed in the way that we do not store anything in Java side and everything is stored in native side. So we should get information from native side when needed.

This is the reason I implemented them in this "wired" way.